src/ServiceFabricGuest.Abstractions/ServiceFabricClientFactory.cs

```diff
@@ -18,6 +19,8 @@ public class ServiceFabricClientFactory
 		/// Creates service fabric client wrapper instance and returns it.
 		/// </summary>
 		public static IServiceFabricClientWrapper Create(ServiceFabricRestClientOptions options) =>
+#pragma warning disable OMEX188 // InitializationCertificateReader using OmexLogger is obsolete. DiagnosticId = "OMEX188"
```

Comment:
This is incorrect.
ServiceFabricClientFactory is required.
InitializationCertificateReader should be updated to use the OTelInitialisationLogger Instance instead.
*at this point, and in this case, it's better to break the old and fix the new.

Reply:
This is pragma to mark InitializationCertificateReader.Instance, not [Obsolete] to ServiceFabricCLientWrapper, so it does not really affect consumer.
OTELInitializationLogger should be available in future PR